src/app/CommandHandlerImpl.cpp

```diff
@@ -566,6 +559,15 @@ Status CommandHandlerImpl::ProcessGroupCommandDataIB(CommandDataIB::Parser & aCo
                 continue;
             }
         }
+
+        if (mpCallback->CommandExists(concretePath) != Status::Success)
```

Comment:
Even though technically speaking this existence check ought to be done the before the ACL check according to spec (in a very general sense, since concurrency or reordering are not relevant if no observable side-effects of responses can arise, such as in the group case), we are doing it after the ACL check to avoid a future where allowing responses from group commands could leak existence.